framework/Element.js

```diff
@@ -44,22 +44,47 @@ export default class Element {
         }
     }
 
-    async click() {
+    async getCoordinates(xOffset, yOffset, elementHandle) {
```

Comment:
I was thinking how this might be used from test, as it will be public. User might expect to call it as element.getCoordinates() and simply get the coordinates of the element. But for this to work we need waiter inside, and we should not pass elementHandle. 
So, this method can have lines inside:
` const elementHandle = await this.wait();
  await elementHandle.focus();`
Then in other methods i.e. click(), these lines can be skipped, as only coordinates is required to do the action. And we know element will be already loaded as this method have already been called.

Reply:
As this is new public method we need to add separate test for it.

Reply:
Addressed.